packages/@atjson/source-url/src/converter.ts

```diff
@@ -84,23 +84,33 @@ URLSource.defineConverterTo(OffsetSource, doc => {
   // Youtube embed code
   // - youtu.be/
   // - youtube-nocookie.com/embed/
+  // - youtube.com/embed
   doc
     .where((url: URLAnnotation) => {
       return (
         url.attributes.host === "youtu.be" ||
-        (url.attributes.host === "www.youtube-nocookie.com" &&
+        (["www.youtube-nocookie.com", "www.youtube.com"].includes(
+          url.attributes.host
+        ) &&
           url.attributes.pathname.startsWith("/embed/"))
       );
     })
     .update((url: URLAnnotation) => {
       let parts = without<string>(url.attributes.pathname.split("/"), "");
       let id = parts.pop();
-      let youtubeURL = `https://www.youtube.com/embed/${id}`;
-      if (url.attributes!.host === "www.youtube-nocookie.com") {
-        youtubeURL = `https://www.youtube-nocookie.com/embed/${id}`;
-      }
+      let youtubeURL =
+        url.attributes!.host === "www.youtube-nocookie.com"
```

Comment:
The definitely defined operator here is no longer required (`attributes!` -> `attributes`)

If it's broken, then it needs to be changed by making the `attribute` type definition to be on the extended annotation base class.